urllib3/util/response.py

```diff
@@ -13,6 +13,12 @@ def is_fp_closed(obj):
     """
 
     try:
+        # Check `isclosed()` first, in case Python3 doesn't set `closed`.
```

Comment:
Sorry, can you update this comment to link to the relevant urllib3 issue as well?